modules/component-store/tsconfig.spec.json

```
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
     "module": "commonjs",
-    "types": ["jest", "node"]
+    "types": ["jest", "node"],
+    "target": "es2020"
```

`target` is already set to `es2020` in the root `tsconfig.json` file. (the same change should be reverted in other `tsconfig.spec.json` files)

```suggestion
    "types": ["jest", "node"]
```